src/include/ceph_features.h

```diff
@@ -137,7 +137,7 @@ DEFINE_CEPH_FEATURE(34, 3, RANGE_BLOCKLIST)
 DEFINE_CEPH_FEATURE(35, 1, OSD_CACHEPOOL)    // 3.14
 DEFINE_CEPH_FEATURE(36, 1, CRUSH_V2)         // 3.14
 DEFINE_CEPH_FEATURE(37, 1, EXPORT_PEER)      // 3.14
-DEFINE_CEPH_FEATURE_RETIRED(38, 1, OSD_ERASURE_CODES, MIMIC, OCTOPUS)
+DEFINE_CEPH_FEATURE(38, 2, CRUSH_MSR)        // X.XX kernel version once in a release
```

Comment:
I don't think so.  A quick grep shows no current users, and clients don't actually have any ec specific behavior that I can think of beyond support for indep steps in crush (which has been part of crush since the beginning).